docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md

```diff
@@ -164,6 +164,28 @@ These examples show two valid YAML syntax options for arrays.
 You must include the `default` namespace in the list if you want the operator to monitor both the `default` namespace and some other namespaces.
 {{% /notice %}}
 
+##### `dns1123Fields`
+
+When the operator processes variable references in the domain resource, such as `$(SERVER_NAME)`,
+it would convert them into DNS-1123 legal values if the variables are referenced
+in fields that requires their values to be conforming to DNS-1123. A default list of such field 
+names can be found inside the class `Legalnames`
+in the `oracle.kubernetes.operator.helpers package`.
+For example, a value `"$(SERVER_NAME)-volume` is specified in a field in a domain resource, 
+where the name of the server is `managed_server1`, and the value is found in a field in the
+list of DNS-1123 field names. Upon variable substitution the operator would replace it with 
+the value `managed-server1-volume` with the underscore, which is not legal in DNS-1123, replaced 
```

Comment:
the value `managed-server1-volume` with the underscore, which is not legal in DNS-1123, replaced with a hyphen. ->
the value `managed-server1-volume`, with underscores, which are not legal in DNS-1123, replaced by hyphens.

Reply:
Hi Rosemary, thanks for the review comments!
Here, the idea was that the underscore in the server name `managed_server1`, which is not legal, will be replaced with a hyphen. Should I rewrite the paragraph as follows to make it less confusing?

For example, a value `$(SERVER_NAME)-volume` is specified in a field in a domain resource that requires DNS-1123. Suppose the name of the server is `managed_server1` which contains an illegal underscore character. Upon variable substitution the operator will replace it with a hyphen, and the resulting value will become `managed-server1-volume`.

Reply:
Yes, that's much more clear. I only suggest that you put a comma after the phrase, "Upon variable substitution." For example, "Upon variable substitution, the operator will replace it with a hyphen, and the resulting value will become managed-server1-volume."

Reply:
Thank you!